### DISCUSSION TITLE
有计划将Masa全家桶发布到Modrinth吗?

### DISCUSSION THREAD

**Discussion by AkarinLiu** [maruohon/malilib#115] [Q&A] [answered]
我一直希望这些模组能够发布到Modrinth，有计划发布吗？
I have been hoping to release these mods to Modrinth, is there any plan to release them?

**Comment by maruohon** [comment, marked as answer]
Yes there are plans to release the mods on Modrinth. I even created a malilib project there a while back already. I just haven't uploaded any mod files or submitted the project for approval yet.

**Reply by manchang666** [reply]
That's great, hope to publish to modrinth soon, so that it will be very convenient for us client authors :)

**Comment by AnemoFlower** [comment]
> 我一直希望这些模组能够发布到Modrinth，有计划发布吗？ I have been hoping to release these mods to Modrinth, is there any plan to release them?

神秘的东方文字
